### DISCUSSION TITLE
Cross-referencing to give native figure numbering in DOCX

### DISCUSSION THREAD

**Discussion by iandol** [quarto-dev/quarto-cli#2335] [Feature Requests]
Quarto has built its own cross-reference filter. One limitation of previous filters which also seems to be shared by Quarto is the inability to generate "native" cross-reference fields in DOCX output. Here are the respective bug reports for pandoc-crossref and pandoc-fignos:

https://github.com/lierdakil/pandoc-crossref/issues/299
https://github.com/tomduck/pandoc-fignos/issues/34

Custom ooxml would be need to be inserted (which is a solution in some other plugins like page break etc.). This would enable e.g. List of Figures using the native fields for figure numbers, and tweaking figure positions in Word would not break the numbering...

As much as I hate it, DOCX remains the dominant format for e.g. publishing scientific articles in many biology/medical/clinical field journals.

**Comment by mcanouil** [comment]
Going back to this.
Did you try latest Pandoc/Quarto? Do you have a reproducible example where we can see the issue?

EDIT: this is actually a duplicate of #2464
